sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/ExpressionTypeCheckingSuite.scala

```diff
@@ -77,12 +77,6 @@ class ExpressionTypeCheckingSuite extends SparkFunSuite with SQLHelper {
   }
 
   test("check types for binary arithmetic") {
-    // We will cast String to Double for binary arithmetic
-    assertSuccess(Add($"intField", $"stringField"))
-    assertSuccess(Subtract($"intField", $"stringField"))
-    assertSuccess(Multiply($"intField", $"stringField"))
-    assertSuccess(Divide($"intField", $"stringField"))
-    assertSuccess(Remainder($"intField", $"stringField"))
```

Comment:
why remove these?

Reply:
Removed by mistake. It is recovered now.